### PR TITLE
Implemented Home Page Layout

### PR DESCRIPTION
**Pull request by faysal-alam-sazim**
### Description of Change
- Created a component named 'Menubar' in 'components' folder
- In 'Menubar' added title of the page, create & clear tasks button and a filter option
- At App.tsx, imported menu bar and created a div for display tasks
- Changed title at index.html
- Deleted assets folder
### Associated Ticket Link:
- https://trello.com/c/l2lcPwZ9
### Related Pull Request:
- https://github.com/faysal-alam-sazim/fancy-todo-app/pull/1
### Screenshots / Screen Recordings: Screen Shot 2024-07-09 at 2 47 14 PM

### CODE REVIEW

File: src/components/MenuBar/MenuBar.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { Group, Text, Button } from "@mantine/core";
+import { IconFilter } from "@tabler/icons-react";
+// import { MantineLogo } from "@mantinex/mantine-logo";
```
Comment (ragib-hossain-sazim) [inline src/components/MenuBar/MenuBar.tsx:3]: Remove commented out code

**Comment by ragib-hossain-sazim** [comment]
Why no screenshot? There should be a screenshot of your layout.

**Comment by ragib-hossain-sazim** [comment]
Please rebase your branch with latest main.

File: src/App.tsx [hunk 2]
```diff
@@ -1,8 +1,14 @@
+import { Text } from "@mantine/core";
+import MenuBar from "./components/MenuBar/MenuBar";
+
 function App() {
   return (
-    <>
-      <h1>Hello World!</h1>
-    </>
+    <div style={{ display: "flex", gap: "20px " }}>
+      <MenuBar></MenuBar>
```
Comment (ishmam-iqbal-sazim) [inline src/App.tsx:7]:
```suggestion
      <MenuBar />
```

If you don't have to pass children to the component, maybe use self closing tags? It looks much cleaner.

**Review by ishmam-iqbal-sazim — APPROVED** [review]
Approved but a general comment for future PRs. Would recommend writing commit messages in the present tense rather than in the past tense as that's what Sazim follows. Check out the commits section in this [doc](https://docs.google.com/document/d/10mowIwsuYMyNkXS7aeL7AJyVNMYVh1vdMf7yn-A6I1k/edit). 

So for example: write `chore(FTDA): Remove unnecessary files` rather than `chore(FTDA): Removed unnecessary files`

**Pull request merged**